src/lib/Toast/Toast.svelte

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+  import { fly } from 'svelte/transition';
+  import { defaultToastProperties, type ToastProperties, type ToastType } from './properties';
+  import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+
+  export let properties: ToastProperties = defaultToastProperties;
+
+  const dispatch = createEventDispatcher();
+
+  let showToast = false;
+  let leftIcon = properties.leftIcon ?? null;
+  let timeout: ReturnType<typeof setInterval>;
+
+  properties = {
+    ...defaultToastProperties,
+    ...properties
+  };
+
+  function hideToast() {
+    showToast = false;
+  }
+
+  function handleAnimationEnd() {
+    dispatch('kill');
+  }
+
+  onMount(() => {
+    showToast = true;
+    timeout = setInterval(hideToast, properties.duration);
+  });
+
+  onDestroy(() => {
+    clearInterval(timeout);
+  });
+
+  function getToastType(type: string): ToastType | null {
+    switch (type) {
+      case 'success':
+        if (!leftIcon) {
+          leftIcon = 'https://sdk.breeze.in/gallery/icons/order-success-icon.svg';
```

Comment:
We should avoid hard coding icons here. 
Users might not want to show icon if `leftIcon` was passed as falsey value.

Reply:
updated.